modules/site-health/admin-site-health.php

```diff
@@ -0,0 +1,248 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Class PLL_Admin_Site_Health to add debug info in WP Site Health
+ *
+ * @since 2.8
+ * @link https://make.wordpress.org/core/2019/04/25/site-health-check-in-5-2/
+ */
+class PLL_Admin_Site_Health {
+
+	public $model;
+
+	/**
+	 * PLL_Admin_Site_Health constructor.
+	 *
+	 * @since 2.8
+	 */
+	public function __construct( &$polylang ) {
+		// Information tab
+		add_filter( 'debug_information', array( $this, 'info_options' ), 15 );
+		add_filter( 'debug_information', array( $this, 'info_languages' ), 16 );
+
+		// tests Tab
+		add_filter( 'site_status_tests', array( $this, 'is_homepage' ) );
+
+		$this->model = &$polylang;
+	}
+
+	/**
+	 * Return a list of key to exclude from site health informations
+	 *
+	 * @return array list of option key to ignore
+	 * @since   2.8
+	 */
+	public function exclude_key() {
+		return array(
+			'flag',
+			'host',
+			'taxonomy',
+			'description',
+			'parent',
+			'filter',
+			'uninstall',
+			'first_activation',
+			'custom_flag',
+		);
+	}
+
+	/**
+	 * Add Polylang Options to Site Health Informations tab.
+	 *
+	 * @param array $debug_info array options to display.
+	 * @return array
+	 * @since   2.8
+	 */
+	public function info_options( $debug_info ) {
+		$options = get_option( 'polylang' );
```

Comment:
This information is stored in the `$polylang->options` property when it is passed to the constructor: https://github.com/polylang/polylang/pull/522/files#diff-cb22506afd9748f751e395fe4f93c2e7R21

I believe it is better to store it in a property and avoid using WordPress functions throughout our code: we'll have more control over the code, we may avoid unnecessary calls to the database, and testing will be made easier.